python/lsst/ts/idl/enums/HVAC.py

```diff
@@ -69,6 +76,37 @@ class DeviceId(IntEnum):
     vex04CargaP04 = 802
 
 
+class DynaleneSafetyState(IntEnum):
+    tmaAlarm = 1
```

Comment:
Our convention is UPPER_SNAKE_CASE for Python enum items.

Would it be practical to make these conform? I am not sure it is, but figured I would ask. I realize it's potentially a headache, due to the useful automatic code generation (and it is annoying that our conventions for enums in ts_xml and Python enum classes don't match).

If you do change these two new enum classes, what about DeviceId, which is pre-existing?

Reply:
I have changed them for `DynaleneSafetyState` and `DynaleneState` but not for `DeviceId` since those are dynamically determined in the HVAC code and changing that would complicate the code further.